draft-ietf-quic-transport.md

```diff
@@ -3407,9 +3407,11 @@ Implementations MUST NOT generate packets that only contain ACK frames in
 response to packets which only contain ACK frames. However, they MUST
 acknowledge packets containing only ACK frames when sending ACK frames in
 response to other packets.  Implementations MUST NOT send more than one packet
-containing only ACK frames per received packet that contains frames other than
-ACK frames.  Packets containing non-ACK frames MUST be acknowledged immediately
-or when a delayed ack timer expires.
+containing only an ACK frame per received packet that contains frames other than
+an ACK frame.  Packets containing non-ACK frames MUST be acknowledged
+immediately or when a delayed ack timer expires. The delayed ack timer should
+never delay an ACK for longer than an RTT, which ensures an ACK frame is sent
```

Comment:
"should never" or "MUST NOT"?

Reply:
Or "does not," if you're making an assertion about the outcome of this strategy.